src/Microsoft.Health.Dicom.Api.UnitTests/Features/Context/DefaultDicomRequestContext.cs

```diff
@@ -26,6 +26,10 @@ public class DefaultDicomRequestContext : IDicomRequestContext
 
     public long BytesTranscoded { get; set; }
 
+    public bool IsRenderedRequested { get; set; }
```

Comment:
You need to add the same in workspaceplatform too

Reply:
Yep, going to cut a release and pull this into workspace platform once this pr gets checked in